# No Title

[Original Post](https://discourse.onlinedegree.iitm.ac.in/t/164277/141)

<p>Hi <a class="mention" href="/u/carlton">@carlton</a> <a class="mention" href="/u/jivraj">@Jivraj</a>,</p>
<p>I followed what you taught in Feb 11 session and tried implementing task A1. My understanding is once i run the subprocess, datagen.py file should be run and /data folder should be created in the project folder. But it is not happening to me. I am getting the following error</p>
<pre><code class="lang-auto">Traceback (most recent call last):
  File "/var/folders/rj/z_3b8hl51558519w90k5hp600000gn/T/datagen4COEF3.py", line 284, in &lt;module&gt;
    os.makedirs(config["root"], exist_ok=True)
    ~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "&lt;frozen os&gt;", line 227, in makedirs
OSError: [Errno 30] Read-only file system: '/data'
</code></pre>
<p>If i can’t automate this process, i don’t see the point writing code for other tasks. Can anyone help me solving this error?</p>